blog/_posts/2015-07-17-Meet-Team-RailsGEnthusiasts.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Introducing Team RailsGEnthusiasts
+layout: blogpost
+created_at: Fri July 17 2015
```

Comment:
Could you change the date here to today? (Sun July 19 2015)
and also in the permalink and the file name?